WMFData/Sources/WMFData/Data Controllers/Shared/WMFPageViewsDataController.swift

```diff
@@ -400,4 +400,13 @@ public final class WMFPageViewsDataController {
 
         return result
     }
+    
+    public func fetchTopCategories(startDate: Date, endDate: Date) async throws -> [String] {
```

Comment:
Can you move this method instead to WMFActivityTabDataController?

Happy to discuss if there's disagreement in architecture but I think of WMFPageViewsDataController and WMFCategoriesDataController as distinct data controllers that operate on particular data sets (page views and categories). I would rather not mix them. They can be referenced from multiple features. Something like WMFActivityTabDataController is more of a feature-specific data controller, it can call shared data controllers (like WMFPageViewsDataController or WMFCategoriesDataController) or fetch directly from the source via services (like WMFBasicService) or stores (like WMFCoreDataStore).